atlassian/jira.py

```diff
@@ -1304,7 +1304,7 @@ def get_project_versions(self, key, expand=None):
             params['expand'] = expand
         return self.get('rest/api/2/project/{}/versions'.format(key), params=params)
 
-    def get_project_versions_paginated(self, key, start=None, limit=None, order_by=None, expand=None):
+    def get_project_versions_paginated(self, key, start=None, limit=None, order_by=None, expand=None, params=None):
```

Comment:
Hi! Could you add the exact arguments, please?
https://docs.atlassian.com/software/jira/docs/api/REST/8.12.1/#api/2/project-getProjectVersionsPaginated